src/openforms/appointments/tests/test_confirmation_emails.py

```diff
@@ -302,3 +303,33 @@ def test_cancel_and_change_instructions(self):
         with self.subTest(type="plain text"):
             self.assertIn(cancel_link, message_text)
             self.assertNotIn(change_link, message_text)
+
+    def test_headers_present_in_confirmation_email(self):
```

Comment:
I think it's better to have this test in the submissions app instead of appointments, since it's not something exclusive or special to appointments - even for non-appointment submissions we expect these confirmation emails (with custom headers) to be sent.